### PR TITLE
Add project map link to README.md

### PR DESCRIPTION
**Pull request by alexkarezin**
**Describe your changes**
Adding a link to the high-level diagrams including module, library dependency and others (https://sourcespy.com/github/bloombergblazingmqsdkjava/). Built directly from source and updated on schedule. Intended to simplify developer's introduction to the project.

**Testing performed**
Documentation-only change.

**Additional context**
In the spirit of transparency - I am the author of the diagrams. Hope contributors find it useful.

### CODE REVIEW

**Comment by 678098** [comment]
Hi @alexkarezin!

We examined the service. Diagrams look good, on par with ones generated by pro-versions of IDEs. The ability to show these without specific local software and even without cloning the repo is indeed a game changer.

For the feedback for the service, I can propose a feature which would make it better than IDEs diagram builders. These builders in common not handling well composition-relation between classes. Examining the code to build these relations, and also clearly visualizing it is a complex task.

We have some concerns however:

- Licensing. It's not fully clear if we can use it. Even if we can, we need to constantly keep track on licensing of this service.

- Security. By placing a badge to a new service, we put some trust in it and show it to the users of our repository. Here are some, but not all possibilities of what could happen. What if at some point the domain name would go to another person, who will do something bad with it. What if the server has vulnerabilities, and some person breaks into it and place phishing links in place of GitHub links to sources. We don't have control over the host and don't have means to secure it. Probably the chances that something bad will happen are small, but the risks we are accepting should be somehow controllable by us and also justified by the impact of the feature.

So in summary we would prefer to keep dependencies at minimum and reject this PR. We are still grateful for your interest in the project. We would gladly accept improvements on static documentation hosted in the repo.

**Comment by alexkarezin** [comment]
Fair points. Yevgeny, thank you for a detailed feedback and idea - it's very much appreciated!
